bundles/org.openhab.binding.tellstick/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -57,4 +57,27 @@
 		</config-description>
 
 	</bridge-type>
+	<bridge-type id="telldus-local">
+		<label>Telldus Local API</label>
+		<description>This bridge represents the Telldus local API.</description>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<label>Local IP Address</label>
+				<description>The local IP address of the Tellstick.</description>
```

Comment:
You could specify `<context>network-address</context>` to get a free validation.

Reply:
Done!